### DISCUSSION TITLE
record drum pattern through computer's keyboard

### DISCUSSION THREAD

**Discussion by samguili** [hydrogen-music/hydrogen#1116] [Hydrogen on Linux]
Hello,

i believe this is a simple matter but i cant figure out how to do it. When i press, let' say, the "V" key, i can hear the floor tom (and all others seems to work fine) but i when press the red record button (as per the manual instructions), nothing happen on the timeline...I am not a very good musician i am afraid and know very little about drums (guitar player here) but i have a rhythm in my head that i would like to record in hydrogen via my pc's keyboard as a single one instrument to be able to edit it after.
Any help greatly appreciated.

specs : ubuntu studio 20.04.1, hydrogen 1.0.0-rc1

Thanks

**Comment by cme** [comment]
Oh, looks like the manual and the spec are out of step here.

To record, you hit the 'record' button, and then the 'play/pause' button.

Much like an '80s tape deck.

I'm not sure if there's a MIDI/Jack-related reason that it works like this, but it's not entirely intuitive. Hmm.

**Comment by samguili** [comment]
That's great, thanks a lot ! It was actually simple but not really user friendly i must agree. I looked on the internets before and strangely enough, there is not so much info about that. Or, maybe, everybody is brighter than me... :)
Cheers !

edit : and thanks for this very quick answer !
